### ISSUE TITLE
Enable image ALT for banner block type

### ISSUE THREAD

**Issue by fuzzbomb** [gareth-fivemile/demo_umami#48]
This has a field called “field_banner_image”. This does not have the ALT text enabled.  We should enable this ALT text, since images here are not likely to be purley decorative.  The example banner on the `/recipes` page certainly needs a text alternative.

The theme currently treats the image as an inline CSS background-image, so it will need a trick to convey the image alternative.

**Comment by fuzzbomb** [comment]
Close this in favour of the new drupal.org issue: [Umami banner block missing alt text](https://www.drupal.org/project/drupal/issues/2938170)
